.goreleaser.yaml

```diff
@@ -173,3 +173,27 @@ dockers:
       - --label=org.opencontainers.image.revision={{ .FullCommit }}
       - --label=org.opencontainers.image.version={{ .Version }}
       - --label=org.opencontainers.image.vendor=Axiom, Inc.
+
+aurs:
+  -
+    name: axiom-bin
+    ids:
+      - nix
+    homepage: https://axiom.co
+    description: Powerful log analytics from the comfort of your command-line
+    maintainers:
+      - 'axiom <hello@axiom.co>'
+    contributors:
+      - 'axiom <hello@axiom.co>'
+    license: MIT
+    private_key: '{{ .Env.AUR_KEY }}'
+    git_url: ssh://aur@aur.archlinux.org/axiom-bin.git
+    skip_upload: auto
+    provides:
+      - axiom-bin
+    conflicts:
+      - axiom-bin
+      - axiom-git
+    commit_author:
+      name: axiom-automation
```

Comment:
I created an AUR account with the `axiomautomation` username already. If this doesn't matter/conflcit there is nothing to do, here.

Reply:
I would imagine it is more just good practice, however as to functionality I'm not sure, better safe than sorry.